### PR TITLE
docs: missing DeviceClass interface

### PR DESCRIPTION
**Pull request by robertsLando**
@AlCalzone Not sure if I imported it correctly

### CODE REVIEW

File: packages/zwave-js/src/lib/test/driver/nodeAsleepNoReject.test.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ describe("regression tests", () => {
 			nodeId: 2,
 			targetValue: 50,
 		});
-		const basicSetPromise2 = driver.sendCommand(command2, {
+		driver.sendCommand(command2, {
```
Comment (robertsLando) [inline packages/zwave-js/src/lib/test/driver/nodeAsleepNoReject.test.ts:61]: @AlCalzone This fix an annoying warning in linter

**Comment by AlCalzone** [comment]
These imports don't do what you're expecting 😅
grafik

I'll need to change the output for classes to not include the implementation.

**Comment by robertsLando** [comment]
> I'll need to change the output for classes to not include the implementation.

In another pr or this?

File: docs/api/node.md [hunk 2]
```diff
@@ -311,7 +311,15 @@ enum InterviewStage {
...
 ```
 
-This property returns the node's [DeviceClass](#DeviceClass-class), which provides further information about the kind of device this node is.
+This property returns the node's **DeviceClass**, which provides further information about the kind of device this node is.
+
+<!-- #import DeviceClass from "zwave-js" -->
+
+<!-- #import BasicDeviceClass from "zwave-js" -->
+
+<!-- #import GenericDeviceClass from "zwave-js" -->
```
Comment (robertsLando) [inline docs/api/node.md]: @AlCalzone Should this be `@zwavejs/config`?

Comment (AlCalzone) [reply docs/api/node.md]: yup, `@zwave-js/config`

Comment (AlCalzone) [reply docs/api/node.md]: > 
> 
> > I'll need to change the output for classes to not include the implementation.
> In another pr or this?

this

**Comment by robertsLando** [comment]
@AlCalzone I also suggest to add an npm script for lint fixing, like `lint:fix` that it's equal to the eslint but with the `--fix` option.

Are you :+1:  for that?

**Comment by AlCalzone** [comment]
> I also suggest to add an npm script for lint fixing

Already possible with `npm run lint:ts -- --fix`. But yeah, why not...

File: docs/index.html [hunk 3]
```diff
@@ -1,64 +1,64 @@
...
-		</style>
-	</head>
-	<body>
-		<div id="app"></div>
-		<script src="//unpkg.com/docsify/lib/docsify.min.js"></script>
-		<script src="//unpkg.com/docsify-edit-on-github/index.js"></script>
-		<!-- Load syntax highlighters -->
-		<script src="//unpkg.com/prismjs/components/prism-bash.min.js"></script>
-		<script src="//unpkg.com/prismjs/components/prism-typescript.min.js"></script>
-		<script src="//unpkg.com/prismjs/components/prism-json.min.js"></script>
```
Comment (AlCalzone) [inline docs/index.html]: doesn't seem like anything changed here - can you reset to the commited version?

File: docs/api/node.md [hunk 4]
```diff
@@ -311,7 +311,15 @@ enum InterviewStage {
...
-This property returns the node's [DeviceClass](#DeviceClass-class), which provides further information about the kind of device this node is.
+This property returns the node's **DeviceClass**, which provides further information about the kind of device this node is.
+
+<!-- #import DeviceClass from "zwave-js" -->
+
+<!-- #import BasicDeviceClass from "@zwave-js/config" -->
+
+<!-- #import GenericDeviceClass from "@zwave-js/config" -->
+
+<!-- #import SpecificDeviceClass from "@zwave-js/config" -->
```
Comment (AlCalzone) [inline docs/api/node.md]: Reminder to self: Change how classes are embedded into the docs

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: docs/index.html (hunk 3 above)
Comment (robertsLando) [reply docs/index.html]: Done

**Review by AlCalzone — DISMISSED** [review]

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**